scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/common/util/SchedulerProxyUserInterface.java

```diff
@@ -611,9 +602,10 @@ public void removeThirdPartyCredential(String key) throws NotConnectedException,
 
     @Override
     public Page<TaskState>  getTaskStates(String taskTag, long from, long to, boolean mytasks,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 10 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)